canonical-web-and-design/ubuntu.com
# Fix rich data training page

## Done

- Fix rich data on training page
- The google search console was signaling this page as failing (see: https://search.google.com/test/rich-results?id=XI6RL64zVxpqUekTbJ4xUQ )

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/training
- Get HTML and paste it on the code tab: https://search.google.com/test/rich-results
- Or get the demo link directly: https://search.google.com/test/rich-results?id=yLUA_7IuL3HSO3R-4gDHsg
